tests/Operation/ModifyCollectionFunctionalTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Operation\ModifyCollection;
+use MongoDB\Operation\CreateCollection;
+use MongoDB\Operation\CreateIndexes;
+
+class ModifyCollectionFunctionalTest extends FunctionalTestCase
+{
+    public function testCollMod()
+    {
+        $operation = new CreateCollection($this->getDatabaseName(), $this->getCollectionName());
+        $operation->execute($this->getPrimaryServer());
+
+        $indexes = [['key' => ['lastAccess' => 1], 'expireAfterSeconds' => 3]];
+        $createIndexes = new CreateIndexes($this->getDatabaseName(), $this->getCollectionName(), $indexes);
+        $createIndexes->execute($this->getPrimaryServer());
+
+        $keyPattern = (object) ['lastAccess' => 1];
+        $modifyCollection = new ModifyCollection(
+                                $this->getDatabaseName(),
```

Comment:
I realize we don't have a code formatting profile yet, but if you'd like to indent a constructor/function call, let's use 4 spaces to indent the arguments and then have `);` on its own line without indentation. For example:

```php
$foo = new Foo(
    'foo',
    'bar'
);
```